linera-views/src/dynamo_db.rs

```diff
@@ -831,6 +831,25 @@ impl DynamoDbStoreInternal {
         }
     }
 
+    async fn test_existence_value_general(
```

Comment:
It would be general in the sense of having a general `key_db` as input.

Yes, we could inline it, but there is more general refactoring to be done here and maybe this can be done in other PRs.